HUD/HideHudNotificationsThisFrame.md

```diff
@@ -0,0 +1,15 @@
+---
+ns: HUD
+aliases: ["0x25F87B30C382FCA7"]
+---
+## _HIDE_HUD_NOTIFICATIONS_THIS_FRAME
+
+```c
+// 0x25F87B30C382FCA7 0x1E63088A
+void _HIDE_HUD_NOTIFICATIONS_THIS_FRAME();
+```
+
+```
```

Comment:
Please remove the \`\`\` code block for the description.

Reply:
Done!